docs/core/tools/dotnet-test.md

```diff
@@ -71,7 +71,13 @@ Where `Microsoft.NET.Test.Sdk` is the test host, `xunit` is the test framework.
 
 - **`--blame-crash`** (Available since .NET 5.0 preview SDK)
 
-  Runs the tests in blame mode and collects a crash dump when the test host exits unexpectedly. This option is only supported on Windows. A directory that contains *procdump.exe* and *procdump64.exe* must be in the PATH or PROCDUMP_PATH environment variable. [Download the tools](/sysinternals/downloads/procdump). Implies `--blame`.
+  Runs the tests in blame mode and collects a crash dump when the test host exits unexpectedly. This option depends on the version of .NET Core used, the type of error and the operating system.
+  
+  For exceptions in managed code, a dump will be automatically collected on .NET 5.0 and newer. It will be generate dump for testhost or any child process that also run on .NET 5.0 and crashed. Crashes in native code will not generate dump. This option works on Windows, MacOS and Linux.
```

Comment:
```suggestion
  For exceptions in managed code, a dump will be automatically collected on .NET 5.0 and later versions. It will generate a dump for testhost or any child process that also ran on .NET 5.0 and crashed. Crashes in native code will not generate a dump. This option works on Windows, macOS, and Linux.
```